src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -0,0 +1,256 @@
+package uk.gov.hmcts.reform.hmc.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.reform.hmc.exceptions.BadRequestException;
+import uk.gov.hmcts.reform.hmc.model.CaseCategory;
+import uk.gov.hmcts.reform.hmc.model.HearingLocation;
+import uk.gov.hmcts.reform.hmc.model.HearingRequest;
+import uk.gov.hmcts.reform.hmc.model.PartyDetails;
+import uk.gov.hmcts.reform.hmc.model.UnavailabilityDow;
+import uk.gov.hmcts.reform.hmc.model.UnavailabilityRanges;
+import uk.gov.hmcts.reform.hmc.utils.TestingUtil;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+class HearingManagementServiceTest {
+
+    @Mock
+    private HearingManagementServiceImpl managementService;
+
+    private HearingManagementServiceImpl hearingManagementService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+        hearingManagementService = new HearingManagementServiceImpl();
+    }
+
+    @Test
+    void shouldFailAsRequestDetailsNotPresent() {
+        HearingRequest hearingRequest = new HearingRequest();
+        hearingRequest.setHearingDetails(TestingUtil.hearingDetails());
+        hearingRequest.setCaseDetails(TestingUtil.caseDetails());
+        try {
+            hearingManagementService.validateHearingRequest(hearingRequest);
+            Assertions.fail("Expected an BadRequestException to be thrown");
+        } catch (Exception exception) {
+            assertEquals("Request details are required", exception.getMessage());
+            assertThat(exception).isInstanceOf(BadRequestException.class);
```

Comment:
For future reference - good to use `assertThrows()` in these kind of tests, which is made for this rather than `try`/`catch` in a test.